app/views/admin/claims/show.html.erb

```diff
@@ -66,6 +66,18 @@
       </table>
     <% end %>
 
+    <% if @inconsistent_claims.any? %>
+      <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive">Warning</span>
+          This claim cannot be approved because <%= @inconsistent_claims.count == 1 ? 'claim' : 'claims' %> <%= @inconsistent_claims.map(&:reference).to_sentence %>
```

Comment:
How about using translations for this? There's a lot of checking counts to work out plurals, and this is supported out of the box with Rails' i18n stuff. For example you could do something like:

```yml
inconsistency_message:
  one: This claim cannot be approved because claim %{reference_numbers} from the same claimant has different values for %{inconsistent_values}
  other: This claim cannot be approved because claims %{reference_numbers} from the same claimant have different values for %{inconsistent_values}
```

You can then use it in the template like so:

```erb
<%= t("inconsistency_message", count: @inconsistent_claims.count, reference_numbers: @inconsistent_claims.map(&:reference).to_sentence, inconsistent_values: @inconsistent_attributes.map { |attribute| Claim.human_attribute_name(attribute).downcase }.to_sentence %>
```

Reply:
I was just about to write what Tekin said about mentioning the month.  I also wonder if there should be some guidance on how they can recover from this. Would it be worthwhile mentioning the service operator playbook?

Reply:
@pezholio Cool! Didn't know that you could use I18n to handle counts. I've changed that and it looks neater.

@titlescreen Yeah, I'm definitely going to get some feedback from a content designer on this. I think that the message could be improved, for sure. It probably would be worth mentioning the playbook, although at the moment we don't really know how service operators are going to handle this situation so there's not going to be anything in the playbook 😄

Reply:
Yeah, it's super useful :)